specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -275,6 +275,65 @@
         "typeProperties"
       ]
     },
+    "BlobEventsTrigger":
+    {
+      "description": "Trigger that runs everytime a Blob event occurs.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MultiplePipelineTrigger"
+        }
+      ],
+      "properties": {
+        "typeProperties":
+        {
+          "description": "Blob Events Trigger properties",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "blobPath": {
+              "description": "path to container, folder, blob, or file extension for which events can trigger the pipeline.",
+              "type": "string"
+            },
+            "events": {
+              "description": "the type of events for which we want to trigger the pipeline.",
+              "$ref": "#/definitions/BlobEventTypes"
+            },
+            "scope": {
+              "description": "the resource id of the Storage Account.",
+              "type": "string"
+            },
+            "maxConcurrency": {

```

Comment:
You might want to add validation for this - should be more than 0 at least, and do you have max for this value?

Reply:
added minimum of 1. Max is configurable, so not specifying.